src/person/views.py

```diff
@@ -76,12 +76,10 @@ def get(self, request):
 
     def post(self, request):
         """Update current authenticated user."""
-        try:
-            data = json.loads(request.body)
-        except json.JSONDecodeError:
-            data = request.data
+        data = request.POST
+        profile_pic = request.FILES.get("image")
```

Comment:
Make sure to update frontend on these new changes.

Reply:
i'll let em know